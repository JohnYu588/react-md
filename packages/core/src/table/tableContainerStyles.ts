import { cnb } from "cnbuilder";

/**
 * @remarks \@since 6.0.0
 */
export interface TableContainerClassNameOptions {
  className?: string;
}

/**
 * @remarks \@since 6.0.0
 */
export function tableContainer(
  options: TableContainerClassNameOptions = {}
): string {
  const { className } = options;

  return cnb("rmd-table-container", className);
}
