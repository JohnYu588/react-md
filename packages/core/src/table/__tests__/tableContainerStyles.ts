import { describe, expect, it } from "@jest/globals";
import { tableContainer } from "../tableContainerStyles.js";

describe("tableContainer", () => {
  it("should be callable without any arguments", () => {
    expect(tableContainer()).toMatchSnapshot();
  });
});
