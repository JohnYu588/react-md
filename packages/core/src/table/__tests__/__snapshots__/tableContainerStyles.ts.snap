// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tableContainer should be callable without any arguments 1`] = `"rmd-table-container"`;
